pkgs/development/python-modules/invocations/default.nix

```diff
@@ -0,0 +1,41 @@
+{ buildPythonPackage
+, blessings
+, fetchFromGitHub
+, invoke
+, lib
+, releases
+, semantic-version
+, tabulate
+, twine
+}:
+
+buildPythonPackage rec {
+  pname = "invocations";
+  version = "2.6.0";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "pyinvoke";
+    repo = pname;
+    rev = version;
+    hash = "sha256-eyOJKVRfn7elyEkERl7hvRTNFmq7O9Pr03lBS6xp0wE=";
+  };
+
+  postPatch = ''
+    substituteInPlace setup.py \
+      --replace "semantic_version>=2.4,<2.7" "semantic_version" \
+      --replace "tabulate==0.7.5" "tabulate"
+  '';
+
+  propagatedBuildInputs = [ blessings invoke releases semantic-version tabulate twine ];
+
+  # There's an error loading the test suite: https://gist.github.com/samuela/ade41eb30b3029b5f4b706d7c012b2bd.
+  doCheck = false;
```

Comment:
```suggestion
  doCheck = false;

  pythonImportsCheck = [
    "invocations"
  ];
```

Reply:
oh good catch! I forgot the `pythonImportsCheck`s.

Reply:
Addressed in https://github.com/NixOS/nixpkgs/commit/82761fb688373e2ca6f08b1f4e718a4f5859eaba and https://github.com/NixOS/nixpkgs/commit/5e7bced95e9e0114653435528c1cdb8e03d48491.